app/components/Company/index.js

```diff
@@ -38,21 +38,29 @@ const Company = props => {
     );
   };
 
-  const renderCompany = () =>
-    company ? (
+  const renderCompany = () => {
```

Comment:
Please make this component a class. Then this method can be the `render` method. `renderSlideShow` will be a method in this class that you call from the `render` method.